services/timers/vmp-mapping/vmp-mapping.timer

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Run vmp-mapping.sh on a regular shedule
+Requires=vmp-mapping.service
+
+[Timer]
+OnCalendar=weekly
```

Comment:
Will this potentially clash with the weekly database update or is the intention to run it beforehand? I don't know if a specific time would be better or if it doesn't actually matter when it runs?